Applications/Resnet/jni/main.cpp

```diff
@@ -136,6 +139,8 @@ std::vector<LayerHandle> createResnet18Graph() {
   layers.push_back(
     createLayer("conv2d", {
                             withKey("name", "conv0"),
+                            withKey("input_shape", "3:32:32"),
```

Comment:
consider using `input_layer` than setting input_shape for a given layer as a recommended practice, as this example will form benchmark for upcoming models.

Reply:
I'll do this in the upcoming PR, I think this will conflict consecutively.

Reply:
alright.